gateway.tf

```diff
@@ -158,6 +159,7 @@ resource "aws_api_gateway_integration_response" "isolation_get_integration_respo
     "method.response.header.Cache-Control"             = "'no-store'",
     "method.response.header.Pragma"                    = "'no-cache'",
     "method.response.header.Strict-Transport-Security" = format("'max-age=%s; includeSubDomains'", var.hsts_max_age)
+    "method.response.header.X-Frame-Options"           = "'SAMEORIGIN'"
```

Comment:
Is this included in the Admin work? this looks like it potentially may break the NI notices of isolation as the IFRAME will be embedded in a different origin.

Reply:
I think we will need to serve this from the backend if I am not wrong.

Reply:
Removed, as should be sent correctly from ECS backend